packages/devtools/src/commands/printPage.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * The print functions are a mechanism to render the document to a paginated format.
+ *
+ * @alias browser.printPage
+ * @see https://w3c.github.io/webdriver/#print
+ * @param {object} options an object of options that affect the pdf output
+ */
+
+import DevToolsDriver from '../devtoolsdriver'
+
+export default async function printPage(this: DevToolsDriver, options: any) {
+    const page = this.getPageHandle()
+    if (!options) {
+        return page.pdf()
+    }
+
+    return page.pdf({
```

Comment:
In order for it to return a string an be compliant with the WebDriver protocol I suggest:

```js
const pdfBuffer = await page.pdf({...})
return pdfBuffer.toString()
```